dev/preview/infrastructure/modules/gce/cloudinit.yaml

```diff
@@ -23,6 +23,6 @@ write_files:
         password = "${dockerhub_passwd}"
       EOF
 
-      sudo systemctl restart containerd.service &
+      sudo systemctl restart containerd.service --wait
```

Comment:
Why was this added?

Reply:
Fixed in 7ff032e540cc479fadc58651213f627837a66c06

Reply:
![image](https://github.com/gitpod-io/gitpod/assets/1272076/a88468f8-27c7-46f0-9ab1-35549775014f)